client/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Hack for LA",
   "license": "GPL-2.0",
   "private": true,
-  "proxy": "http://localhost:5000",
+  "proxy": "https://foodoasis.la",
```

Comment:
The proxy setting allows us to run both server and client locally. This change should not be merged.